GsaGH/Helpers/GsaAPI/Mappings.cs

```diff
@@ -211,6 +211,26 @@ internal static class Mappings {
         { "Torsional", typeof(TorsionalSpringProperty) }
       };
 
+    internal static readonly IList<DiagramTypeMapping> _diagramTypeMappingDisplacement
+      = new List<DiagramTypeMapping>() {
+            new DiagramTypeMapping("Translation Ux", DiagramType.TranslationUx,
+              ApiDiagramType.TranslationUx),
+            new DiagramTypeMapping("Translation Uy", DiagramType.TranslationUy,
+              ApiDiagramType.TranslationUy),
+            new DiagramTypeMapping("Translation Uz", DiagramType.TranslationUz,
+              ApiDiagramType.TranslationUz),
+            new DiagramTypeMapping("Resolved Translation U", DiagramType.ResolvedTranslationU,
```

Comment:
can we shorten this? "Res. Translation U"
please?

Reply:
done